index.md

```diff
@@ -6,32 +6,32 @@ description: "Beemo is a free antispam Discord Bot that can stop userbot raids a
 permalink: /
 ---
 
-# Beemo
+# What is Beemo?
 {: .fs-9 }
 
-Beemo is a free antispam Discord Bot that can stop userbot raids against your server.
+Well, Beemo is an antispam Discord Bot built to stop userbot raids.
 
 {: .fs-6 .fw-300 }
 
 ## Important Documentation Notes
 
-The documentation for Beemo Max is not yet complete, but there are some pages available to give a general idea. Since Beemo is designed to be configured in-Discord, all of the commands are generally well documented via their respective help menus. If you would like more information on Beemo Max beyond what is present in the docs, [join the Discord](https://beemo.gg/Discord)!
+The documentation for Beemo Max is limited, but there are some pages available. Since Beemo is designed to be configured in-Discord, all of the commands are generally well documented via their respective help menus. If you would like more information on Beemo Max (beyond the docs), [join the Discord](https://beemo.gg/Discord)!
 
 ## Adding Beemo & Setting Up 
 
-Adding Beemo to your server is super simple and takes seconds. 
+Adding Beemo to your server is super simple. 
```

Comment:
```suggestion
Adding Beemo to your server is super simple!
```
This is not actually on your changes, but rather the original docs, and I'm taking your PR as an opportunity to improve it. I think the `!` adds extra expressiveness 😊